integrated_channels/degreed/admin/__init__.py

```diff
@@ -106,7 +104,6 @@ def force_content_metadata_transmission(self, request, obj):
     force_content_metadata_transmission.label = "Force content metadata transmission"
 
 
-@admin.register(DegreedLearnerDataTransmissionAudit)
```

Comment:
Is there any plan to remove Degrred classes from the code?

Reply:
I think this is the first step towards removing Degreed v1 code, disabling their configurations and transmissions.